mica/archive/aca_dark/dark_cal.py

```diff
@@ -30,21 +30,34 @@ def date_to_dark_id(date):
     """
     Convert ``date`` to the corresponding YYYYDOY format for a dark cal identifiers.
 
-    :param date: any DateTime compatible format
+    :param date: any CxoTime compatible format
     :returns: dark id (YYYYDOY)
     """
-    date = DateTime(date).date
-    return date[:4] + date[5:8]
+    t = CxoTime(date)
```

Comment:
No 1-letter variable names in new code. You can use `tm` as a standard "time", but in general always use descriptive variable names.